{{cookiecutter.project_slug}}/.envs/.local/.postgres

```diff
@@ -0,0 +1,2 @@
+POSTGRES_USER={{cookiecutter.project_slug}}
+POSTGRES_PASSWORD=postgrespassword
```

Comment:
I wonder if it would make sense to pre-populate these with random strings? Just like the secret key?

Reply:
This one's addressed.

Reply:
Gotta also ensure the generated keys match `POSTGRES_PASSWORD` format (it took Travis a few tries before finishing green, and I guess it might be the case with PostgreSQL failing to create `POSTGRES_USER` with the generated `POSTGRES_PASSWORD`).